client/src/components/Payment/PaymentListForm/PaymentListForm.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import PaymentCard from '../PaymentCard/PaymentCard';
+import ModalButton from '../../Modal/ModalButton/ModalButton';
+
+export default function PaymentListForm(): JSX.Element {
+  const [listPaymentMethods, setListPaymentMethods] = useState(<div>Loading...</div>);
+  useEffect(() => {
+    fetch('/payment/list-payment-methods', {
+      method: 'GET',
+    }).then((res) => {
+      res.json().then((data) => {
+        console.log(data);
```

Comment:
We can remove this console log since it's not needed here

Reply:
Removed